client/src/app/components/meiern/meiern.component.html

```diff
@@ -1,8 +1,18 @@
 <mat-card class="meiern-card">
+  <button class="backButton" [matTooltip]="'WIP (Dont use it PLS)'" (click)="back()"> <mat-icon>keyboard_backspace</mat-icon></button>
```

Comment:
Du kannst hier wahrscheinlich einen RouterLink benutzen
```suggestion
  <a routerLink="/" class="backButton" [matTooltip]="'WIP (Dont use it PLS)'" > <mat-icon>keyboard_backspace</mat-icon></a>
```
So irgendwie sollte das funktionieren.

Reply:
ahhh routerlink hieß dess :D